packages/kit/src/exports/vite/index.js

```diff
@@ -375,7 +375,7 @@ function kit({ svelte_config }) {
 					const { assets, base } = svelte_config.kit.paths;
 
 					if (browser) {
-						return `export const base = ${s(base)};
+						return `export const base = ${global}.base;
 export const assets = ${global}.assets;`;
```

Comment:
If `paths.base` and `paths.assets` are both unspecified then I think `asset_expression` in `render.js` will be identical to `base_expression`. Should we omit it in that case and so something like this?

```suggestion
export const assets = ${global}.assets ?? base;`;
```

Reply:
implemented this change